docs/integrations/aa/alchemy-account-kit.md

```diff
@@ -66,15 +66,15 @@ values={[
 <TabItem value="npm">
 
 ```js
-npm i @lit-protocol/pkp-ethers@cayenne
+npm i @lit-protocol/pkp-ethers@datil-dev
```

Comment:
```suggestion
npm i @lit-protocol/pkp-ethers
```

Version `6.1.0` of the SDK (that supports Datil-dev) is not installed from NPM by default